README.md

```diff
@@ -1,22 +1,22 @@
 gulp-sww
 ========
 
-A [gulp](http://gulpjs.com/) plugin that will automatically generate all the files and infrastructure to make your web app work offline with ServiceWorkers.
+A [gulp](http://gulpjs.com/) plugin to make your web app work offline.
 
 Details
 -------
-This plugin uses the library [ServiceWorkerWare](https://github.com/fxos-components/serviceworkerware) to provide the integration with ServiceWorker.
+This plugin uses the library [ServiceWorkerWare](https://github.com/fxos-components/serviceworkerware) to provide the integration with Service Worker.
+The AppCache manifest is generated by [node-appcache-generator](https://github.com/arcturus/node-appcache-generator).
```

Comment:
We can explain that appcache will be used as a fallback if your browser doesn't support service workers